packages/slate-react/src/components/leaf.tsx

```diff
@@ -43,7 +43,13 @@ const Leaf = (props: {
   }, [placeholderRef])
 
   let children = (
-    <String isLast={isLast} leaf={leaf} parent={parent} text={text} />
+    <String
+      key={Math.random()}
```

Comment:
Can you use a number key like `let key = 0` at the beginning of the file, then `key++` here instead? Makes it more standard and easier to augment in the future if we need.

Reply:
OK, I have add comments for it too